### ISSUE TITLE
bug `indentation` with empty lines between multiple values in property

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#895]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Throw error about incorrectly indentation if empty line present between multiples values of property.

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```scss
a {
    box-shadow:
        0 0 0 1px #5b9dd9,

        0 0 2px 1px rgba(30, 140, 190, 0.8);
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "indentation": 4
  }
}
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Nothing, or may be create rule for allow/disallow empty lines between multiple values of property. If need rule i create separate issue about this problem.

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.scss
4:1     Expected indentation of 8 spaces (indentation)
```

**Comment by davidtheclark** [comment]
Fixed released in v5.
